invenio_rdm_records/services/review/service.py

```diff
@@ -12,17 +12,23 @@
 from invenio_drafts_resources.services.records import RecordService
 from invenio_i18n import lazy_gettext as _
 from invenio_records_resources.services.uow import (
+    NotificationOp,
     RecordCommitOp,
     RecordIndexOp,
     unit_of_work,
 )
-from invenio_requests import current_request_type_registry, current_requests_service
+from invenio_requests import (
+    current_request_type_registry,
+    current_requests_service,
+)
 from invenio_requests.resolvers.registry import ResolverRegistry
+from invenio_users_resources.records.api import UserAggregate
 from marshmallow import ValidationError
 
 from invenio_rdm_records.proxies import current_rdm_records
```

Comment:
why do we have relative and absolute imports? I think this was already like this but I think these should be made relative.